packages/cli-server-api/package.json

```diff
@@ -1,21 +1,20 @@
 {
   "name": "@react-native-community/cli-server-api",
-  "version": "16.0.2",
+  "version": "13.5.1",
   "license": "MIT",
   "main": "build/index.js",
   "publishConfig": {
     "access": "public"
   },
   "dependencies": {
-    "@react-native-community/cli-debugger-ui": "16.0.2",
-    "@react-native-community/cli-tools": "16.0.2",
+    "@react-native-community/cli-tools": "13.5.1",
     "compression": "^1.7.1",
     "connect": "^3.6.5",
     "errorhandler": "^1.5.1",
     "nocache": "^3.0.1",
     "pretty-format": "^26.6.2",
     "serve-static": "^1.13.1",
-    "ws": "^6.2.3"
```

Comment:
IIRC we needed to have the same version of this package as in `react-native` - is it still true?